game-core/src/test/java/games/strategy/triplea/delegate/power/calculator/TotalPowerAndTotalRollsTest.java

```diff
@@ -36,8 +33,6 @@
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.triplea.java.collections.IntegerMap;
-import org.triplea.util.Triple;
-import org.triplea.util.Tuple;
 
 @ExtendWith(MockitoExtension.class)
 class TotalPowerAndTotalRollsTest {
```

Comment:
In a later PR, I'll split this test apart so that the AA tests are in `AaPowerStrengthAndRollsTest` and the non-AA are in `PowerStrengthAndRollsTest`.  I didn't do it as part of this PR because I wanted to be able to show that I didn't break any of the existing tests.